### PR TITLE
Add proper README and flowchart with tech tree

### PR DESCRIPTION
**Pull request by cit1zen**

### CODE REVIEW

**Comment by argentlike** [comment]
Here Adam, probably need to explain it.  I don’t have Ryan’s contact details.
The concept is we have a base logic of streams and dependencies (research, buildings etc) that we then allow applications to be applied to.


regards,
dee

dee swain life pathways
Business Coach | Life Coach | Motivation | Wellbeing | Lecturer Teacher

+61(0)424155353 I dee@swain.com.au<mailto:dee@swain.com.au>

File: README.md [hunk 1]
```diff
@@ -1,87 +1,58 @@
 # Community tech tree for Workers & Resources
 
+[![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
 
-## Instalation
+A community tech tree replaces the Workers & Resources vanilla tech tree
+with one that is bigger, better, and managed by the modders community.
+The tree allows modders and content creators to lock their buildings
+behind the research in a scalable and synchronized way.
```
Comment (ryantheskinny) [inline README.md]: remove "the"

File: README.md [hunk 2]
```diff
@@ -1,87 +1,58 @@
...
-$COST 700
-
-$AVAILABLE
-
-$UNLOCK_RESEARCH engineering_1
-$UNLOCK_BUILDING mechanical_components_factory
-
-$NAME 11100
-$DESC 11101
-
```
Comment (ryantheskinny) [inline README.md]: can we use $NAME_STR "insert name" here?

File: README.md [hunk 3]
```diff
@@ -1,87 +1,58 @@
...
-$NAME 11102
-$DESC 11103
+## Installation
 
-$RESEARCH_ADD
+1. Download the repository:
+   1. by cloning repository using GIT,
+   2. or downloading a [ZIP file][3].
+2. Merge `research` directory with the game's `research` directory.
+   Path to this directory is ``.
```
Comment (ryantheskinny) [inline README.md]: C:\Program Files (x86)\Steam\steamapps\common\SovietRepublic\media_soviet\research

File: README.md (hunk 2 above)
Comment (cit1zen) [reply README.md]: We can't :( This is also in an outdated version of the README.

**Pull request merged**